src/build_mozc.py

```diff
@@ -370,18 +371,40 @@ def WriteEnvironmentFile(path, env):
     f.write(nul.join(entries).encode('utf-8'))
 
 
-def UpdateEnvironmentFilesForWindows(out_dir):
+def UpdateEnvironmentFilesForWindows(out_dir, vcvarsall_path):
```

Comment:
`vcvarsall_path` looks unused.

Reply:
Ah, this should have been passed to `get_vs_env_vars` as `vcvarsall_path_hint` parameter at the line 403.

Fixed.